src/components/ProjectCard/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import BaseText from "../BaseText";
+import Label from "../Label";
+import "./ProjectCard.css";
+
+function ProjectCard({ title, description, label1, label2, label3, label4 }) {
```

Comment:
@AbdurrahmanSogbesan im just seeing this. You need to pass the labels as a prop in an array. You cant hardcode it like this because we cant control how many labels may be present.